frontend/javascripts/oxalis/view/novel_user_experiences/01-present-modern-controls.js

```diff
@@ -10,7 +10,7 @@ export default function PresentModernControls() {
   const dispatch = useDispatch();
   const activeUser = useSelector(state => state.activeUser);
   const [isModalVisible, setIsModalVisible] = React.useState(
-    activeUser != null && !activeUser.novelUserExperienceInfos.hasSeenModernControlsModal,
+    activeUser != null && !activeUser.novelUserExperienceInfos.shouldSeeModernControlsModal,
```

Comment:
I think this is why I currently see the modal even when signing up on the dev instance.

```suggestion
    activeUser != null && activeUser.novelUserExperienceInfos.shouldSeeModernControlsModal,
```